flox-bash/lib/commands/general.sh

```diff
@@ -181,7 +181,14 @@ function floxSearch() {
 			shift
 			;;
 		# TODO: Deprecate `-v,--verbose'
-		-l|--long|-v|--verbose)
+		-v|--verbose)
+			_msg="the flag 'flox search -v,--verbose' is deprecated."
```

Comment:
I think this PR fixes also https://github.com/flox/flox-private/issues/376 so this PR receives bonus points :)